Gong updates to Handbook. (#6903)
fleetdm/fleet: handbook/people/README.md

We have excluded anyone with an email domain from @cooley.com or @formationfinancial.com from Gong's recording feature. These are professional services firms working with Fleet on internal matters, and calls with them are considered internal.  

Our goal in using Gong and recording calls is to capture insights from sales, customer, and community meetings and improve how we position and sell our product. We never intend to make anyone uncomfortable, and we hope you reach out to our DRI for Gong, Nathan Holliday, or Mike McNeil if you have questions or concerns.  

> It is highly encouraged that anyone using Gong go through the Gong Academy training for the appropriate role at [https://academy.gong.io/](https://academy.gong.io/).

Reviewer: I'd replace the "highly encouraged" part here and replace it with "If you need help with Gong, go through"

It doesn't make sense for most folks to watch this.

Author: This is done.

We configure our Zoom meetings to let participants join before the host starts the meeting. We do this to make sure meetings start on time, even if the host isn't there.

### Gong

> Capturing video from meetings with customers, propsects, and community members outside the company is an important part of building world-class sales and customer success teams, and is a widespread practice across the industry.

Reviewer 2: ```suggestion
Capturing video from meetings with customers, prospects, and community members outside the company is an important part of building world-class sales and customer success teams and is a widespread practice across the industry.
```

At Fleet, we use Gong to capture Zoom meetings and share them company-wide if a team member with a Gong license attends certain meetings, generally those with at least one person from outside of Fleet in attendance.  

> While some other Fleeties may have a Gong seat if it becomes necessary in the course of their work, the typical use case at Fleet is for employees on the company's sales, customer success, or customer support teams. You should be notified anytime you join a recorded call with an audio message announcing "this meeting is being recorded" or "recording in progress".  To stop a recording, the person on the call with a Gong seat can press "Stop". If the call has external participants and is being recorded, this call is stored in Gong for future use. 

Reviewer 2: ```suggestion
While some other Fleeties may have a Gong seat if it becomes necessary in their work, the typical use case at Fleet is for employees on the company's sales, customer success, or customer support teams. You should be notified anytime you join a recorded call with an audio message announcing "this meeting is being recorded" or "recording in progress."  To stop a recording, the person on the call with a Gong seat can press "Stop." If the call has external participants and is recorded, this call is stored in Gong for future use. 
```

> While some other Fleeties may have a Gong seat if it becomes necessary in the course of their work, the typical use case will be Fleet employees on the company's sales, customer success, or customer support teams. You will be notified anytime you join a recorded call with an audio message announcing that "this meeting is being recorded" or "recording in progress".  To stop a recording, the person on the call with a Gong seat can press "Stop". If the call has external participants and is being recorded, this call will end up being stored in Gong for future use. 

Reviewer: @hollidayn We can stick to present tense, rather than future, here and throughout, since everything is in draft and this is live today.

> To access a recording saved in Gong, visit [app.gong.io](app.gong.io) and sign in with SSO.  Everyone at Fleet has access, whether they have a Gong seat or not, and you can explore and search through any call transcripts that have been uploaded unless they have been marked as private (though the best practice would be to not record any calls you don't want captured).  Should you ever make a mistake, and you need to delete something, you can delete the video in Gong or reach out to Nathan Holliday or Mike McNeil for help and we will delete it immediately, without watching the video.  Note that any recording stopped with 60 seconds of the start of the recording is not saved in Gong and there will be no saved record of it.  There is a chance that some calls may still be saved in Zoom, and to counter this Nathan Holliday will periodically delete any recordings found in Zoom's storage without viewing them.

Reviewer 2: ```suggestion
To access a recording saved in Gong, visit [app.gong.io](app.gong.io) and sign in with SSO. Everyone at Fleet has access, whether they have a Gong seat or not, and you can explore and search through any uploaded call transcripts unless someone marks them as private (though the best practice would be not to record any calls you don't want to be captured). If you ever make a mistake and need to delete something, you can delete the video in Gong or reach out to Nathan Holliday or Mike McNeil for help. They will delete it immediately without watching the video. Note that any recording stopped within 60 seconds of the start of the recording is not saved in Gong, and there will be no saved record of it. There is a chance that Gong still may save some in Zoom. To counter this, Nathan Holliday will periodically delete all recordings found in Zoom's storage without viewing them.
```

> Most folks at Fleet should see no difference in their meetings if they aren't interfacing with external parties.  For those with a Gong seat, or scheduling a call with someone in attendance that has a Gong seat, and who do not wish for a their Zoom call with an external party to be recorded, the following words and phrases in the title will disable the Gong recording for the call:

Reviewer 2: ```suggestion
Most folks at Fleet should see no difference in their meetings if they aren't interfacing with external parties. For those with a Gong seat or scheduling a call with someone in attendance that has a Gong seat who does not wish for their Zoom call with an external party to record, the following words and phrases in the title will disable the Gong recording for the call:
```

> 1 on 1

Reviewer: I'd frame this list as examples of keywords, since we'll no doubt update them over time, and point out how someone can see the list or request a change (e.g. ask in #digital-experience, or create an issue, put in current release in digex board, and assign nathan, or whatever you think is easiest)

Author: Ok, I'll see if the list is accessible within Gong by standard users and unlicensed collaborators.  If not - I'll ask you about a Plan B.

1 : 1 
one on one
confidential
privileged
interview
sensitive
private
1:1
1-1
1x1
1<>1
one-on-one
internal
no shadows
[no shadows]

> We have also excluded anyone with an email domain from @cooley.com or @formationfinancial.com from Gong's recording feature as these are professional services firms working with Fleet on internal matters and calls with them are considered internal calls.  

Reviewer 2: ```suggestion
We have excluded anyone with an email domain from @cooley.com or @formationfinancial.com from Gong's recording feature. These are professional services firms working with Fleet on internal matters, and calls with them are considered internal.  
```

> Our goal in using Gong and recording calls is to capture insights from sales, customer, and community meetings, and to improve the way we position and sell our product.  It is never our intent to make anyone uncomfortable, and we hope you reach out to either our DRI for Gong, Nathan Holliday, or Mike McNeil if you have questions or concerns.  

Reviewer 2: ```suggestion
Our goal in using Gong and recording calls is to capture insights from sales, customer, and community meetings and improve how we position and sell our product. We never intend to make anyone uncomfortable, and we hope you reach out to our DRI for Gong, Nathan Holliday, or Mike McNeil if you have questions or concerns.  
```
